test/alignak_test.py

```diff
@@ -136,102 +137,92 @@ class AlignakTest(unittest.TestCase):
-        if len(self.conf.arbiters) == 1:
-            arbdaemon = Arbiter([''], [''], False, False, None, None)
-
-            arbdaemon.load_modules_manager()
-
-            # we request the instances without them being *started*
-            # (for those that are concerned ("external" modules):
-            # we will *start* these instances after we have been daemonized (if requested)
-            me = None
-            for arb in self.conf.arbiters:
-                me = arb
-                arbdaemon.do_load_modules(arb.modules)
-                arbdaemon.load_modules_configuration_objects(raw_objects)
-
-        self.conf.create_objects(raw_objects)
-        self.conf.instance_id = 0
-        self.conf.instance_name = 'test'
-        # Hack push_flavor, that is set by the dispatcher
-        self.conf.push_flavor = 0
-        self.conf.load_triggers()
-        #import pdb;pdb.set_trace()
-        self.conf.linkify_templates()
-        #import pdb;pdb.set_trace()
-        self.conf.apply_inheritance()
-        #import pdb;pdb.set_trace()
-        self.conf.explode()
-        #print "Aconf.services has %d elements" % len(self.conf.services)
-        self.conf.apply_implicit_inheritance()
-        self.conf.fill_default()
-        self.conf.remove_templates()
-        #print "conf.services has %d elements" % len(self.conf.services)
-        self.conf.override_properties()
-        self.conf.linkify()
-        self.conf.apply_dependencies()
-        self.conf.explode_global_conf()
-        self.conf.propagate_timezone_option()
-        self.conf.create_business_rules()
-        self.conf.create_business_rules_dependencies()
-        self.conf.is_correct()
-        if not self.conf.conf_is_correct:
-            print "The conf is not correct, I stop here"
-            self.conf.dump()
-            return
-        self.conf.clean()
-
-        self.confs = self.conf.cut_into_parts()
-        self.conf.prepare_for_sending()
-        self.conf.show_errors()
-        self.dispatcher = Dispatcher(self.conf, self.me)
-
-        scheddaemon = Alignak(None, False, False, False, None)
-        self.scheddaemon = scheddaemon
-        self.sched = scheddaemon.sched
-        scheddaemon.load_modules_manager()
-        # Remember to clean the logs we just created before launching tests
-        self.clear_logs()
-        m = MacroResolver()
-        m.init(self.conf)
-        self.sched.load_conf(self.conf)
-        e = ExternalCommandManager(self.conf, 'applyer')
-        self.sched.external_command = e
-        e.load_scheduler(self.sched)
-        e2 = ExternalCommandManager(self.conf, 'dispatcher')
-        e2.load_arbiter(self)
-        self.external_command_dispatcher = e2
-        self.sched.conf.accept_passive_unknown_check_results = False
-
-        self.sched.schedule()
+    @classmethod
+    def setUp(cls):
+        """
+        Load default configuration when start test
+
+        @verified
```

Comment:
I modify functions in alignak_test.py to be ok and it's annotation for the mment to see if the function is ok or not. At the end I will delete this line